### ISSUE TITLE
alt text in \bmlDescription is absent in embedded SVGs

### ISSUE THREAD

**Issue by xworld21** [vlmantova/bookml#16]
When using the native LaTeXML TikZ support,
```latex
\begin{tikzpicture}
...
\end{tikzpicture}
\bmlDescription{Alt text}
```
fails to add the alt text to the image.
